File: 251010/배수의 개수/number-of-multipliers.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int array[10];
    int count1 = 0;
    int count2 = 0;

    for(int i = 0; i < 10; i++) {
        cin >> array[i];
    }

    for(int i = 0; i < 10; i++) {
        if (array[i] % 3 == 0) {
            count1 += 1;
        } 
        
        if (array[i] % 5 == 0) {
            count2 += 1;
        }
    }

    cout << count1 << " " << count2;
    return 0;
}

File: 251028/배수의 개수/number-of-multipliers.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int a = 0; 
        int count1 = 0;
        int count2 = 0;

        for(int i = 0; i < 10; i++) {
            a = int.Parse(Console.ReadLine());
            if(a % 3 == 0 && a % 5 == 0) {
                count1++;
                count2++;
            } else if(a % 3 == 0) {
                count1++;
            } else if(a % 5 == 0) {
                count2++;
            }
        }
        Console.Write(count1 + " " + count2);
    }
}
